### PR TITLE
Remove outdated references

### PR DESCRIPTION
**Pull request by Ada-Codina**
See: https://github.com/aad-for-linux/aad-for-linux/issues/7

### CODE REVIEW

File: auth_aad.c [hunk 1]
```diff
@@ -6,7 +6,7 @@
  *             packet compression.
  *
  *  Copyright (C) 2002-2018 OpenVPN Inc <sales@openvpn.net>
- *  Copyright (C) 2019 CyberNinjas <info@cyberninjas.com>
+ *  Copyright (C) 2019 Infinite Omicron, LLC. <open.source@infinite-omicron.com>
```
Comment (oxr463) [inline auth_aad.c:9]:
```suggestion
 *  Copyright (C) 2021 Infinite Omicron, LLC. <open.source@infinite-omicron.com>
```

File: debian/changelog [hunk 2]
```diff
@@ -2,10 +2,10 @@ openvpn-auth-aad (0.0.2-1) UNRELEASED; urgency=medium
...
 
- -- Lucas Ramage <lramage@cyberninjas.com>  Thu, 15 Aug 2019 16:01:23 +0000
+ -- Infinite Omicron, LLC. <open.source@infinite-omicron.com>  Thu, 15 Aug 2019 16:01:23 +0000
 
 openvpn-auth-aad (0.0.1-1) UNRELEASED; urgency=medium
 
   * Initial release.
 
- -- Lucas Ramage <lramage@cyberninjas.com>  Wed, 19 Jun 2019 18:07:48 +0000
+ -- Infinite Omicron, LLC. <open.source@infinite-omicron.com>  Wed, 19 Jun 2019 18:07:48 +0000
```
Comment (oxr463) [inline debian/changelog:11]: We don't need to change the changelog entries.

File: debian/copyright [hunk 3]
```diff
@@ -1,10 +1,10 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: openvpn-auth-aad
-Upstream-Contact: Lucas Ramage <lramage@cyberninjas.com>
-Source: https://github.com/CyberNinjas/openvpn-auth-aad
+Upstream-Contact: Infinite Omicron, LLC. <open.source@infinite-omicron.com>
+Source: https://github.com/aad-for-linux/openvpn-auth-aad
 
 Files: *
-Copyright: 2019 CyberNinjas <info@cyberninjas.com>
+Copyright: 2019 Infinite Omicron, LLC. <open.source@infinite-omicron.com>
```
Comment (oxr463) [inline debian/copyright:7]:
```suggestion
Copyright: 2021 Infinite Omicron, LLC. <open.source@infinite-omicron.com>
```

**Review by oxr463 — CHANGES_REQUESTED** [review]
Looks like we need a few changes. But first, can we rebase on master? Good work!

Reference(s):
- https://git-scm.com/book/en/v2/Git-Branching-Rebasing

**Review by Ada-Codina — COMMENTED** [review]
reviewed

**Pull request closed without merging**